pkg/plugins/trivy/plugin_test.go

```diff
@@ -2201,110 +2201,110 @@ default ignore = false`,
 						TerminationMessagePolicy: corev1.TerminationMessageFallbackToLogsOnError,
 						Env: []corev1.EnvVar{
 							{
-								Name: "HTTP_PROXY",
+								Name: "TRIVY_SEVERITY",
 								ValueFrom: &corev1.EnvVarSource{
 									ConfigMapKeyRef: &corev1.ConfigMapKeySelector{
 										LocalObjectReference: corev1.LocalObjectReference{
 											Name: "trivy-operator-trivy-config",
 										},
-										Key:      "trivy.httpProxy",
+										Key:      "trivy.severity",
 										Optional: ptr.To[bool](true),
 									},
 								},
 							},
 							{
-								Name: "HTTPS_PROXY",
+								Name: "TRIVY_IGNORE_UNFIXED",
 								ValueFrom: &corev1.EnvVarSource{
 									ConfigMapKeyRef: &corev1.ConfigMapKeySelector{
 										LocalObjectReference: corev1.LocalObjectReference{
 											Name: "trivy-operator-trivy-config",
 										},
-										Key:      "trivy.httpsProxy",
+										Key:      "trivy.ignoreUnfixed",
 										Optional: ptr.To[bool](true),
 									},
```

Comment:
why did the ordering of these test cases change? are you using a different linter?